resources/user-agents/crawlers/data-mining/data-mining.json

```diff
@@ -43,6 +43,34 @@
           "properties": {
             "Browser": "CommonCrawler Node"
           }
+        },
+        {
+          "match": "Mozilla/5.0 (compatible; Dataprovider/*",
+          "properties": {
+            "Browser": "Dataprovider Bot",
+            "Browser_Maker": "Dataprovider.com"
+          }
+        },
+        {
+          "match": "Mozilla/5.0 (compatible; um-LN/*",
+          "properties": {
+            "Browser": "UberMetrics Bot",
```

Comment:
Thers is already a file for bot from this company (crawlers/general-crawlers/ubermetrics.json). Please move this pattern to this file and maybe move that file to this folder.